api/src/main/java/org/openmrs/api/context/ServiceContext.java

```diff
@@ -205,6 +206,20 @@ public ObsService getObsService() {
 		return getService(ObsService.class);
 	}
 	
+	/**
+	 * @return condition related service
+	 */
+	public ConditionService getConditionService() {
```

Comment:
Can you also add the @since annotations to these two methods?

Reply:
Okay.